docs/config/cloud.md

```diff
@@ -73,3 +63,31 @@ exports.config = {
   }]
 };
 ```
+## Reporter
+We have a specific SauceLabs reporter that can add test details like describe() and it() blocks annotations in the commands execution log. Please note that SauceLabs reporter depends on SauceLabs environment and so will break if running with plain browser. 
```

Comment:
this info is already in the 'test annotations' section - better to rename it and combine the info.

Reply:
merged both sections